requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
 black
 check-manifest
 codespell
-docker-compose
+docker-compose # Not available in py3.10
```

Comment:
So, to do local testing with py3.10, we'd have to install docker-compose separately (in the system, rather than via conda)?

Reply:
Yes sir. 😬

Reply:
I'll stick to doing local testing with Py 3.9